group_vars/bibdata/production.yml

```diff
@@ -104,9 +100,9 @@ rails_app_vars:
   - name: AWS_REGION
     value: "us-east-1"
   - name: AWS_ACCESS_KEY_ID
-    value: "{{ vault_scsb_s3_access_key_local }}"
+    value: "{{ vault_aws_access_key }}"
```

Comment:
Do we have only a single set of AWS access and secret access keys for all of PUL? If not, we might want to include `bibdata` in these var names so we don't get confused across projects.